week2/w2_q1.cpp

```diff
@@ -0,0 +1,33 @@
+//CodeU: Assignment2-Q1
+//
+//  Created by Julia Semavina on 24.05.2017
+//
+//  problem specification:
+//      Given a Binary Tree and a key, write a function that prints all the ancestors of the key
+//      in the given binary tree.
+//
+//  comments:
+//      Data Structures: Node ("Node.h"), BinaryTree (BinaryTree.h")
+//      input data:
+//          follow the instructions
+//      output data:
+//          all the ancestors of the vertex, started with parent
+//      ShowAncestors function contains a solution
+//
+
+#include <iostream>
+#include "w2_BinaryTree.h"
+
+
+void ShowAncestors(BinaryTree &MyBTree, int key) {
```

Comment:
Also, is good to add const to ensure your clients that the tree won't be modified.
This may cascade and you may need to also make 'find' a const function.